docs/installation.rst

```diff
@@ -20,6 +20,35 @@ There are different ways to install |project_name|:
     have the latest code or use revision control (each release is
     tagged).
 
+.. _installation-requirements:
+
+Pre-Installation Considerations
+-------------------------------
+
+Repository Filesystem
+~~~~~~~~~~~~~~~~~~~~~
+
+- Filenames may contain any valid ascii character and be of any length. |project_name|
+  stores data only 3 directory levels deep and uses short directory and filenames.
+- |project_name| requires only read/write permissions within the repository filesystem.
+- Source files are split up into pieces called "chunks". During the chunking
+  process, small files will typically yield one chunk while larger files will
+  be broken up into chunks of ~2MiB in size.
+- Chunks are combined into segment files. The number and size of segment files
+  can be set in the :ref:`config-file`.
```

Comment:
|project_name| stores backup metadata and data into so-called segment files.
The target size of these files and also the count of these files per directory is set in the :ref:`config-file`.